tex/variational-dating.tex

```diff
@@ -11,15 +11,158 @@
+with possible long introgressed segments.
+For use cases typical to population genetics,
+these assumptions are not typically true,
+and more sophisticated use of the available information is necessary to get good estimates.
+
+The currently most-used ARG inference methods are tsinfer \cite{XXX}, Relate \cite{XXX}, ARGweaver \cite{XXX}, and ARGneedle \cite{XXX}.
+We focus in this paper on the first two,
+as ARGweaver does not scale to large numbers of samples (although it samples from the full posterior on ARGs),
+and ARG-Needle was only recently released.
+(However, the methods we present here could be applied to any ARG,
+as long as it is convertible to tree sequence format \citep{ts}.)
+These two provide an informative comparison, as -- roughly speaking --
+Relate is closer to a tree-by-tree method,
+while tsinfer is closer to a ``haplotype-copying'' approach \citep{li-and-stephens}.
+
+These two aspects of the data -- ``trees and haplotypes'' --
+create a fundamental tension that various methods navigate in different ways.
+On the one hand, the relationship between sampled genotypes at each location on the genome is described by a single tree,
+inference of trees is well-understood, and trees are very efficient data structures.
+On the other hand, nearby trees share a good deal of structure,
+so inferring a sequence of independent trees leaves out a great deal of information.
+Each ancestor in an ARG can be thought of as a node is a sequence of trees covering a portion of the genome,
+or as an ancestral haplotype that different individuals may have inherited different portions of.
+To get a sense for this, note that as one moves along the genome, the longest branches in a tree relating $n$ samples
+are expected to be of order $n^2$ times as long as the shortest ones.
+This implies that as one moves along the genome,
+the oldest parts of the tree (which tend to be longest) will change roughly $n^2$ times as fast as the youngest parts of the tree.
+Said another way, the longest shared haplotypes tend to be around $n^2$ times as long as the shortest ones.
+This is particularly relevant for the dating problem:
+the extent to which ancestral haplotypes in an inferred ARG extend along the genome
+(i.e., the amount of sharing of node identity between trees)
+determines how much information from many trees can be used to inform the dating of each individual node.
+This information includes both hard constraints
+(e.g., a given ancestor cannot be older than any ancestor it inherits from in any tree)
+and soft constraints
+(e.g., mutations at a given genomic location may inform the dates of many ancestors,
+including whose haplotypes don't cover that location).
+
+Our goal in this paper is to improve the accuracy by which the ages of ancestors in an ARG are inferred,
+while maintaining sufficient computational speed that the method can be applied to Biobank-scale samples.
```

Comment:
Mention uncertainty, cost of quantifying uncertainty (e.g. requires MCMC), variational methods solve